tests/model_fields/test_uuid.py

```diff
@@ -35,6 +35,14 @@ def test_null_handling(self):
         self.assertEqual(loaded.field, None)
 
 
+class TestMigrations(TestCase):
```

Comment:
I wonder if it would be feasible to raise a warning or an exception if database queries are ran during a `SimpleTestCase`.